src/port/Game.cpp

```diff
@@ -87,8 +77,6 @@ Cup* gStarCup;
 Cup* gSpecialCup;
 Cup* gBattleCup;
 
-ModelLoader gModelLoader;
-
 HarbourMastersIntro gMenuIntro;
 
 Editor::Editor gEditor;
```

Comment:
Please also remove the commented out block at the end of CustomEngineInit()

<img width="724" height="307" alt="image" src="https://github.com/user-attachments/assets/633aa1d0-dffd-46a4-ab7a-3e9a837f6c05" />

Reply:
done